include/mbedtls/pk.h

```diff
@@ -202,7 +202,7 @@ typedef struct mbedtls_pk_rsassa_pss_options {
 
 /* Internal helper to define which fields in the pk_context structure below
  * should be used for EC keys: legacy ecp_keypair or the raw (PSA friendly)
- * format. It should be noticed that this only affect how data is stored, not
+ * format. It should be noted that this only affect how data is stored, not
```

Comment:
Sorry, not trying to be extra picky but one thing was missed: "affect" --> "affects". It needs the "s" on the end.

(I've done this the old way since I don't understand the pro tip from the last round of review...